otter.ts

```diff
@@ -25,10 +24,24 @@ export interface SpeechViewSummary {
     displayId: string
 }
 
+export function getExportTemplate(): string {
+    var exportTemplateConfigVar = process.env.exportTemplate?.trim()
+    if (!exportTemplateConfigVar) {
```

Comment:
thir can be collapsed into something like `return exportTemplateConfigVar || "default value"